octoprint_simple_filament_change_buttons/static/js/simple_filament_change_buttons.js

```diff
@@ -11,29 +11,70 @@ $(function() {
 		// assign the injected parameters
 		self.controlViewModel = parameters[0];
 
+		self.unloadFilament = function() {
+			cmds = [
+				// Heat to specified temp
+				"M109 S" + self.temperature(),
+				// Do unload
+				"M702",
+				// Cool hotend
+				"M104 S0"
```

Comment:
If possible, it should be restoring the previous temperature after load/unload - not setting it to 0

Reply:
Done